pkg/utils/protoutils/marshal.go

```diff
@@ -98,6 +99,19 @@ func MarshalMapEmitZeroValues(from resources.Resource) (map[string]interface{},
 	return m, err
 }
 
+func MarshalMapFromProto(from proto.Message) (map[string]interface{}, error) {
+	out := &bytes.Buffer{}
+	if err := jsonpbMarshaler.Marshal(out, from); err != nil {
+		panic(fmt.Sprintf("failed to marshal proto to bytes: %v", err))
```

Comment:
Why are we panicking here and not just returning the error and leaving it up to the caller. If this is intended we should remove the `error` return value and call this function `MustMarshalMapFromProto`

Reply:
Oh good point, I missed this when moving the logic into this util function. Will fix.